webservices/cxf/pom.xml

```diff
@@ -179,6 +179,16 @@
             <version>2.0.31-beta</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+                        <groupId>OKI</groupId>
+                        <artifactId>OkiOSID</artifactId>
+                        <version>2.0</version>
+                </dependency>
+        <dependency>
+            <groupId>OKI</groupId>
+            <artifactId>OkiSID</artifactId>
+            <version>rc6.1</version>
+        </dependency>
```

Comment:
Consider to include <scope>test</scope> for both dependencies in that case, and also remove version from OkiOSID artifact to inherit master pom version.